### ISSUE TITLE
Make simple bias bar charts with existing jsons

### ISSUE THREAD

**Issue by gleckler1** [PCMDI/pcmdi_metrics#349]
PG and maybe JW will try to get this one in time for v1.1

**Comment by durack1** [comment]
@gleckler1 I also promised to contribute to this..

**Comment by doutriaux1** [comment]
@chaosphere2112 has a bar graph/histogram graphic method

**Comment by chaosphere2112** [comment]
Yup, I'm working on getting it into UV-CDAT (pull request will be issued shortly)

**Comment by durack1** [comment]
@chaosphere2112 @doutriaux1 things are currently being developed using matplotlib.. Once it's available if you guys are happy to assist in pulling matplotlib -> VCS/UV-CDAT for the plots that would be ideal

**Comment by gleckler1** [comment]
@lee1043 If you start this before leaving go ahead and start a new branch for it.   Taylor diagram is higher priority.   But that other thing you have going on is much higher priority than all of this!

**Comment by lee1043** [comment]
@gleckler1 Thanks, I will. :)

**Comment by lee1043** [comment]
Draft script using matplotlib was upload to branch 349_bias_bar_chart. Still need to give correct unit information for the y-axis, which should coming from the json file although it is not in there for now...

**Comment by gleckler1** [comment]
@lee1043 Do we still need this branch or is the latest bar chart covered in another branch (enso, monsoon, variability)?   Not sure if we should have a general (multiple purpose) branch for bar charts or if it should be scattered about different branches.
